examples_utils/benchmarks/command_utils.py

```diff
@@ -120,6 +120,33 @@ def get_benchmark_variants(benchmark_name: str, benchmark_dict: dict) -> list:
     return variants
 
 
+def remove_wandb_args(cmd: str) -> str:
+    """Remove all wandb args and their values from command strings.
+    """
+
+    new_arg_list = []
+    arg_list = shlex.split(cmd)
+    
+    skip = False
+    for i in len(arg_list):
+        if "--" in arg_list[i] and "wandb" in arg_list[i]:
+            # Also remove the values given to wandb args using a skip bool for
+            # next iteration
+            if not "-" in arg_list[i+1][:2]:
```

Comment:
think we should check if i + 1 < len(arg_list), if --wandb arg is put right at the end of the command

Reply:
Thanks, yes addressed in last comment above